src/node/nodestate.h

```diff
@@ -200,7 +200,8 @@ namespace ccf
 
```

Comment:
It seems `quote_max_size` can be removed now? Since the new OE API gives us a report of whatever size it wishes. Then we can construct `quote` from the report data after calling `oe_get_report`, rather than trying to size here and `assign`ing later.

Reply:
Done